CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file. See [Keep a
 CHANGELOG](http://keepachangelog.com/) for how to update this file. This project
 adheres to [Semantic Versioning](http://semver.org/).
 
+### Changed
+- Replace Poison with Jason for JSON encoding.
+- Reduce the log level used for dev mode warning.
+- Update dependenices.
+
+### Fixed
+- Fix crashes caused by json encoding error, we now print an error on stdout
```

Comment:
Technically we write to the log when there is an encoding error, which may or may not be logging to the console. Also, as a tiny nitpick it says `json` here but `JSON` in the line above.